addons/sys_attenuate/fnc_toggleDebugInfo.sqf

```diff
@@ -0,0 +1,154 @@
+
+            {
+                private _attenuation = getNumber(_attConfig >> _x);
+                if (_attenuation > 0) then {
+                    _hintMsg pushBack (format ["<t align='left' font='PuristaMedium'> %2 = %3",  _rootAtt select [11], _x select [11], _attenuation]);
+                    
+                    if (_forEachIndex == (_compartmentsCount - 1)) then {
+                        _hintMsg pushBack "</t>";
+                    } else {
+                        _hintMsg pushBack "    |  </t>";
+                    };
+                };
+            } forEach _compartments;
+            _hintMsg pushBack "<br/>";
+        } forEach _compartments;
+
+        _compartmentsTurnedOut = ["Compartment1","Compartment2","Compartment3","Compartment4"] apply {
+            if (isClass (_config >> "ACRE" >> "attenuationTurnedOut" >> _x)) then {
+                _x
+            } else {
+                ""
+            };
+        };
+        _compartmentsTurnedOut = _compartmentsTurnedOut - [""];
+
+        private _compartmentsTurnedOutCount = count _compartmentsTurnedOut;
+        if (_compartmentsTurnedOutCount > 0) then {
+            _hintMsg pushBack (format["<t align='left' size='1.15' color='#0051a1' font='PuristaMedium'>%1</t><br/>", "Turned Out"]);
+        };
+
+        {
+            private _rootAtt = _x;
+            private _attConfig = _config >> "ACRE" >> "attenuationTurnedOut" >> _rootAtt;
+
+            _hintMsg pushBack (format ["<t align='left' size='1.15' color='#68bbff' font='PuristaMedium'>%1</t><br/>", _x]);
+
+            {
+                private _attenuation = getNumber(_attConfig >> _x);
+                if (_attenuation > 0) then {
+                    _hintMsg pushBack (format ["<t align='left' font='PuristaMedium'> %2 = %3",  _rootAtt select [11], _x select [11], _attenuation]);
+                    
+                    if (_forEachIndex == (_compartmentsCount - 1)) then {
+                        _hintMsg pushBack "</t>";
+                    } else {
+                        _hintMsg pushBack "    |  </t>";
+                    };
+                };
+            } forEach _compartmentsTurnedOut;
+            _hintMsg pushBack "<br/>";
+        } forEach _compartmentsTurnedOut;
+
+        hintSilent parseText (_hintMsg joinString "");
+
+        // Draw attenuation values for all speakers
+        {
+            private _dir = (getDir (missionNamespace getVariable ["BIS_DEBUG_CAM", acre_player])) - (getDir _x);
+            private _atten = [_x] call acre_sys_attenuate_fnc_getUnitAttenuate;
+            private _color = [0, 1 - _atten, _atten, _atten + .6];
+            private _text = format ["%1: %2", (name _x) select [0,10], _atten];
+            drawIcon3D ["\a3\ui_f\data\gui\cfg\Hints\icon_text\group_1_ca.paa", _color, _x modelToWorldVisual [0,0,0], 1, 1, _dir, _text, 1, 0.04, "RobotoCondensed"];
```

Comment:
Ain't paths in SQF always case-sensitive on Linux ?
```suggestion
            drawIcon3D ["\a3\ui_f\data\GUI\Cfg\Hints\icon_text\group_1_ca.paa", _color, _x modelToWorldVisual [0,0,0], 1, 1, _dir, _text, 1, 0.04, "RobotoCondensed"];
```

Reply:
Ignore the commit msg, I'm doing too many things at once atm 😆